apps/guardian-ui/src/languages/en.json

```diff
@@ -70,12 +71,18 @@
       "no-gateways-info-title": "No connected gateways yet!",
       "no-gateways-info-description": "Lightning node operators can connect to your federation to provide Lightning Network interoperability. Once connected, they will appear here."
     },
+    "api-announcements": {
+      "label": "API Announcements",
+      "guardian": "Guardian",
+      "api-url": "API URL",
+      "revision": "Revision"
+    },
     "config": {
       "label": "Federation Config",
       "view-config": "View Config",
       "missing-meta-module": "Editing Meta fields is not possible. The Meta module is not available for this federation.",
       "manage-meta": {
-        "tab-label": "Manage Meta",
+        "label": "Manage Meta",
```

Comment:
since this is a key change you should probably update the other language files but as long as no code is using the unused key this shouldn't break anything